src/drivers/distance_sensor/sf1xx/sf1xx.cpp

```diff
@@ -637,26 +635,74 @@ namespace sf1xx
 
 SF1XX	*g_dev;
 
-void	start(uint8_t rotation);
-void	stop();
-void	test();
-void	reset();
-void	info();
+int bus_options[] = {
```

Comment:
@bkueng for example in src/lib/drivers/device/i2c.h? (in this case I should rename it since this array exists in some drivers).. or should I create a new i2c_distance.h just common to all distance sensors?

Reply:
> @bkueng for example in src/lib/drivers/device/i2c.h?

yes, and it wil need to have a better name. But it can be used by all I2C drivers.